pycufsm/preprocess.py

```diff
@@ -364,22 +364,91 @@ def template_out_to_in(sect):
         rad = 0
     else:
         rad = sect['r_out'] - thick/2
-    depth = sect['d'] - thick/2 - rad - rad - thick/2
+    depth = sect['h'] - thick/2 - rad - rad - thick/2
 
-    if sect['l'] == 0:
+    if sect['d'] == 0:
         b_1 = b_1 - rad - thick/2
         l_1 = 0
     else:
         b_1 = b_1 - rad - thick/2 - (rad + thick/2)*np.tan(np.pi/4)
-        l_1 = sect['l'] - (rad + thick/2)*np.tan(np.pi/4)
-    if sect['l'] == 0:
+        l_1 = sect['d_1'] - (rad + thick/2)*np.tan(np.pi/4)
+    if sect['d'] == 0:
         b_2 = b_2 - rad - thick/2
         l_2 = 0
     else:
         b_2 = b_2 - rad - thick/2 - (rad + thick/2)*np.tan(np.pi/4)
-        l_2 = sect['l'] - (rad + thick/2)*np.tan(np.pi/4)
+        l_2 = sect['d_2'] - (rad + thick/2)*np.tan(np.pi/4)
```

Comment:
a) I know CUFSM labeled lip length as `d` - but most manufacturer catalogs call it `l` and the internal variables are all `l`. I strongly suggest we use `l` and not `d`.
b) If you're assuming the lip length is defined as `l_1` and `l_2`, then the conditionals need to be on `l_1` and `l_2` - not just `l`.

Reply:
I couldn't find any place where the template_out_to_in and template_calc functions were being used..
I made the changes while trying to make a Jupyter Notebook which could load SFIA sections to analyze.
Will change it back to 'l'

Reply:
![image](https://user-images.githubusercontent.com/66215806/95353828-baab4a00-0891-11eb-9590-94e858b96b91.png)

I used ipywidgets package to make this template..Since you already wrote the template_out_to_in and template_calc in the preprocess, getting this to work was a lot easier..sorry about changing 'l' to 'd'..I was trying to understand the code comparing it to the matlab version so changed it to 'd'

Reply:
No worries! Experimenting is needed to get things to work properly. The screenshot there looks nice! But yeah, since I was the one porting the code initially, I took a couple liberties to clean up things that didn't make sense - like always indicating the minor axis with `z` (it was randomly `y` in a couple places), and not using `d` for the lip length. :-)